std/complex.d

```diff
@@ -280,26 +280,42 @@ if (isFloatingPoint!T)
     Complex!(CommonType!(T, R)) opBinaryRight(string op, R)(const R r) const
         if (op == "/" && isNumeric!R)
     {
-        import core.math : fabs;
-        typeof(return) w = void;
-        if (fabs(re) < fabs(im))
+        version (FastMath)
```

Comment:
Do we want to accommodate the new version as well as the old one? If so, would this require a spec mention?

Reply:
The fastmath version is less accurate as no range reduction is done.

Reply:
Ok, but how is the user supposed to choose between the 2? Shouldn't we add a mention in the changelog or in the spec about this?